http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -373,13 +376,12 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       hasDeletePermission = listOfPermissions.toSet.contains(AppAction.DeleteApp)
       _ <- if (hasDeletePermission) F.unit else F.raiseError[Unit](ForbiddenError(userInfo.userEmail))
 
-      canDelete = AppStatus.deletableStatuses.contains(appResult.app.status)
-      _ <-
-        if (canDelete) F.unit
-        else
-          F.raiseError[Unit](
-            AppCannotBeDeletedException(cloudContext, appName, appResult.app.status, ctx.traceId)
-          )
+      // Galaxy is the only app that has custom deletion logic that requires `helm uninstall`. For all other apps, we'll
```

Comment:
Nice 👍

Reply:
One suggestion though would be to move this comment into the `checkIfCanBeDeleted` function, because the current status is that we are not yet attempting to delete non galaxy apps by deleting the namespace

Reply:
I'll keep this in mind in my future PR...I'll have a few more PRs coming :)